src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,84 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+/**
+ * A class that represent deadline task
+ */
+public class Deadline extends Task {
+    private String name;
+    private boolean isDone;
+    private String time;
+
+    /**
+     * constructor
+     * @param name
+     * @param isDone
+     * @param time
+     */
+    public Deadline(String name, boolean isDone, String time) {
+        super();
+        this.name = name;
+        this.isDone = isDone;
+        this.time = time;
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean isDone() {
+        return isDone;
+    }
+
+
+    /**
+     * get time
+     * @return
+     */
+    public String getTime() {
+        String[] s = this.time.split(" ");
+
+        LocalDate date = LocalDate.parse(s[1]);
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(s[0] + " ");
+        stringBuilder.append(date.getMonth() + " ");
+        stringBuilder.append(date.getDayOfMonth() + " ");
+        stringBuilder.append(date.getYear() + " ");
+        for (int i = 2; i < s.length; i++) {
+            stringBuilder.append(s[i] + " ");
+        }
+
+        return stringBuilder.toString();
+    }
+
+    /**
+     * search whether keyword exist
+     * @param keyword
+     * @return
+     */
+    public boolean searchKeyword(String keyword) {
```

Comment:
Could make this sound more boolean? 
```suggestion
    public boolean hasKeyword(String keyword) {
```